Uspihh.Models/EntityModels/GroupEntity.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Uspihh.Models.EntityModels
+{
+    public class GroupEntity
+    {
+        public long GroupId { get; set; }
+        public string GroupName { get; set; }
+        public int MaxCountOfStudent { get; set; }
```

Comment:
The same issue with DataType
Why int?
V1.1_Table_Group.sql -> [MaxCountOfStudents] [smallint]]
smallint in db is the same as short in C#